console/src/app/directives/copy-to-clipboard/copy-to-clipboard.directive.ts

```diff
@@ -7,7 +7,7 @@ export class CopyToClipboardDirective {
   @Input() valueToCopy: string = '';
   @Output() copiedValue: EventEmitter<string> = new EventEmitter();
 
-  @HostListener('click', ['$event']) onMouseEnter($event: any): void {
+  @HostListener('click', ['$event']) onClick($event: any): void {
```

Comment:
Is this a wrong merge conflict resolution? @peintnermax what is correct, here?

Reply:
this could also be a random value cause its just a variable.. onClick definitively makes more sense 👍